warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/error/ErrorDataTypeHandler.java

```diff
@@ -117,9 +118,9 @@
     // Initialize a default (hash-based) UID builder
     private UIDBuilder<UID> uidBuilder = UID.builder();
     
-    public static final Text EVENT_COLF = new Text("e");
+    public static final Text EVENT_COLF = new Text(MetadataFamilyConstants.E);
     public static final Text INFO_COLF = new Text("info");
```

Comment:
Why no constant for this one?

Reply:
It's a WIP.
Also decided to start with the biggest offenders first.  This one is seldom used.